experimental/cc_slurm_nhc/cc_slurm_nhc/specs/default/cluster-init/files/azure_nccl_allreduce.nhc

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+  
+# Check for NVlink ssues by running NCCL allreduce.
```

Comment:
Since the file name is not too descriptive, I would add here that this runs on a single node through NVLink instead of IB